README.md

```diff
@@ -70,18 +70,19 @@ Supported Compilers
+| CUDA (nvcc)                                                                        | :white_check_mark: <br/> (CUDA 11.0-11.5) | :white_check_mark: <br/> (CUDA 11.0-11.5) | :white_check_mark: <br/> (CUDA 11.0-11.5) | :white_check_mark: <br/> (CUDA 11.5) | :x:                    | :x:                     | :white_check_mark: <br/> (CUDA 11.0-11.5) | :white_check_mark: <br/> (CUDA 11.1-11.5) | :white_check_mark: <br/> (CUDA 11.4-11.5) | :white_check_mark: <br/> (CUDA 11.4-11.5) | -                       | :x:                                                   | :white_check_mark: <br/> (CUDA 11.2-11.5) |
+| CUDA (clang)                                                                       | -                                         | -                                         | -                                         | -                                    | -                      | -                       | -                                         | :x:                                       | :white_check_mark: <br/> (CUDA 9.2-10.1)  | :white_check_mark: <br/> (CUDA 10.0-10.2) | -                       | -                                                     | -                                         |
+| [HIP-4.0.1](https://alpaka.readthedocs.io/en/latest/install/HIP.html) (clang)      | :x:                                       | :x:                                       | :x:                                       | :x:                                  | :x:                    | :x:                     | :x:                                       | :x:                                       | :x:                                       | :white_check_mark:                        | -                       | -                                                     | -                                         |
+| [HIP-4.3 and 4.5](https://alpaka.readthedocs.io/en/latest/install/HIP.html) (clang)| :x:                                       | :x:                                       | :x:                                       | :x:                                  | :x:                    | :x:                     | :x:                                       | :x:                                       | :x:                                       | :x:                                       | :white_check_mark:      | -                                                     | -                                         |                           
```

Comment:
I guess it was too noisy to have one HIP line?
And what about HIP 4.2 and 4.4? Why are those versions missing?

Reply:
ohh the first line should be HIP 4.2 and the HIP 4.3 and 4.5 line is separated because the clang compiler differs.
HIP 4.4 is never officially released, AMD skipped this number.

Reply:
fixed